tools/development/helpers/install-python.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+################################################################################################################################################################
+
+# @project        Open Space Toolkit ▸ Physics
+# @file           tools/development/helpers/install-python.sh
+# @author         Lucas Brémond <lucas@loftorbital.com>
+# @license        Apache License 2.0
+
+################################################################################################################################################################
+
+PYTHON_VERSION="3.9"
+
+################################################################################################################################################################
+
+project_directory="$(git rev-parse --show-toplevel)"
+python_directory="${project_directory}/build/bindings/python/OpenSpaceToolkitAstrodynamicsPy-python-package-${PYTHON_VERSION}"
+
+pushd "${python_directory}" > /dev/null
+
+pip install .
```

Comment:
add `--force-reinstall` to be stateless when rebuilding in between

Reply:
@Derollez like this? `pip install . --force-reinstall`